data_source/gateway_test.go

```diff
@@ -0,0 +1,103 @@
+package dataSource
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/NethermindEth/juno/clients"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAdaptStateUpdate(t *testing.T) {
+	jsonData := []byte(`{
```

Comment:
As mentioned above, we should add a reference to where this data is gotten from

Reply:
Some are created by hand and rest is either taken from gateway with some manual changes or verbatim. Either way I dont have access to the URLs anymore.